src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageResponseCommentUiTest.java

```diff
@@ -188,4 +195,68 @@ private InstructorFeedbackResultsPage loginToInstructorFeedbackResultsPage(Strin
                 .withSessionName(testData.feedbackSessions.get(fsName).getFeedbackSessionName());
         return loginAdminToPage(resultsUrl, InstructorFeedbackResultsPage.class);
     }
+
+    private void addEmptyCommentToResponseAndCheckStatusMessage(String commentModelId) {
+        // TODO: `clickCommentModalButton` should wait for modal to open before returning
+        resultsPage.clickCommentModalButton(commentModelId);
+        resultsPage.addFeedbackResponseCommentInCommentModal("showResponseCommentAddForm" + commentModelId, "");
+        // TODO: all instances of this should do an immediate check instead of waiting for the error message
+        resultsPage.waitForCommentFormErrorMessageEquals(commentModelId,
+                Const.StatusMessages.FEEDBACK_RESPONSE_COMMENT_EMPTY);
+        resultsPage.closeCommentModal(commentModelId);
+    }
+
+    // Add first comment to response
+    private void addCommentToValidResponseAndVerify(String commentModelId) {
+        String commentId = commentModelId + "-1";
+        resultsPage.clickCommentModalButton(commentModelId);
+        // TODO: Wait for comment to be added before returning
+        resultsPage.addFeedbackResponseCommentInCommentModal("showResponseCommentAddForm" + commentModelId,
+                "test comment 1");
+        // TODO: all instances of this should do an immediate check instead of waiting for the comment to be added.
+        resultsPage.waitForFeedbackResponseCommentAdded(commentId, "test comment 1", "Teammates Test");
+        resultsPage.closeCommentModal(commentModelId);
+    }
+
+    private void editFirstCommentOnResponseAndVerify(String commentModelId, String commentText) {
+        String commentId = commentModelId + "-1";
+        resultsPage.clickCommentModalButton(commentModelId);
+        resultsPage.editFeedbackResponseCommentInOpenedCommentModal(commentId, commentText);
+        resultsPage.waitForFeedbackResponseCommentAdded(commentId, commentText, "Teammates Test");
+        resultsPage.closeCommentModal(commentModelId);
```

Comment:
`resultsPage.waitForFeedbackResponseCommentAdded(commentId, commentText, "Teammates Test");` This line verifies it. 

This was also renamed/ refactored in https://github.com/TEAMMATES/teammates/pull/8859/files. Shall I rename it to `verifyAddedFeedbackResponseComment`?

Reply:
This is due to PR #8859.

The function actually *waits* and *verifies*. In the review loop it was renamed appropriately first. 

https://github.com/TEAMMATES/teammates/pull/8859#discussion_r194254891

And then since we added the TODO that is first waits and then verifies, the function got renamed to something that might cause doubts that it doesn’t verify.

https://github.com/TEAMMATES/teammates/pull/8859#discussion_r200828936

Maybe we can rename this function to show that it waits and verifies.

Maybe `waitAndVerifyFeedbackResponseCommentAdded`
Since its a trivial change, maybe we can do it in this PR itself?